### PR TITLE
Improve wording

### PR DESCRIPTION
**Pull request by AlvaroBurnett**

### CODE REVIEW

File: content/Video_Display.md [hunk 1]
```diff
@@ -738,7 +738,7 @@ Mode 0 - H-Blank Period
 Mode 1 - V-Blank Period
 ```
 
-Aside from that, OAM can be accessed at any time by using the DMA
+During those modes, OAM can be accessed at any time by using the DMA
```
Comment (ISSOtm) [inline content/Video_Display.md:741]: I'd further clarify that OAM DMA out-prioritizes the PPU in accessing OAM, and that the PPU will read `$FF` from OAM during that time. I'd thus make it a separate paragraph, to make it stand out.

**Comment by AlvaroBurnett** [comment]
I incorporated your suggestion. Please check if it's correct.

**Review by ISSOtm — APPROVED** [review]

**Review by avivace — APPROVED** [review]

**Comment by avivace** [comment]
Thanks a lot for all these contributions, @AlvaroBurnett !

**Comment by AlvaroBurnett** [comment]
> Thanks a lot for all these contributions, @AlvaroBurnett !

I'm happy to be useful :)

**Pull request merged**